scripts/README.md

```diff
@@ -95,6 +96,36 @@ Options:
   --help           Show this message and exit.
 ```
 
+## report-outdated-by-mod.py
+
+```
+$ python ./scripts/report-outdated-by-mod.py --help
+Usage: report-outdated-by-mod.py [-h] [--path PATH] [target_lang ...]
+
+    This script compares markdown files across different language directories
+    to identify and report localized documents that may be outdated,
+    based on modification date differences.
+
+    It focuses primarily on:
+    1. Reporting outdated documents based on modification date differences.
```

Comment:
It would be great if these statistics could also be published somewhere on the web periodically so that people could consult the progress status and be motivated to catch up. Maybe not the scope of this PR but an idea ;)

Reply:
Thank you, @ricardoamaro. 
Yes, if there is a general agreement among the localization teams, we could potentially automate this program to run periodically (using GitHub workflows, etc.). 
Once this PR is merged, I plan to introduce this tool to the localization contributors and test its usefulness.